src/parser/parser.spec.ts

```diff
@@ -675,7 +675,8 @@ describe('fabric.Parser', () => {
   it('getCssRule', () => {
     expect(getCSSRules).toBeDefined();
     const rules: Record<PropertyKey, any> = {};
-    const doc = getFabricDocument();
+    // NOTE: We need to use a new fresh document here because vitest in browser mode already adds some stylesheets which pollutes the test
+    const doc = getFabricDocument().implementation.createHTMLDocument('');
```

Comment:
in vitest browser mode there are already some stylesheets imported to testing frame by vitest, so I had to get clean doc implementation here
works fine in both jsdom and chromium

Reply:
This test i think is wrong.
The parser and getCSSRules is used for SVGdocuments, i think here to test the function we assumed that one document is like anohter.
getFabricDocument return the document that holds the javascript from fabricjs.
We need a generic document, if we have a generic api that works in both cases like `new Document()` or `new XLMDocument()` (since is svg) we should prefer that.

Reply:
I guess this is then something for a separate PR as a todo?

Reply:
What i meant is, without having to create it from the fabric document itself.
Is there any global we can invoke that give us a new document, empty?

Reply:
yes i just cheked `document.implementation` is just a normal browser thing, i just never used it.